tools/http_benchmark.py

```diff
@@ -16,17 +17,28 @@ def deno_http_benchmark(deno_exe):
     return run(deno_cmd)
 
 
-def node_http_benchmark(deno_exe):
+def node_http_benchmark():
     node_cmd = ["node", "tools/node_http.js", ADDR.split(":")[1]]
     print "http_benchmark testing NODE."
     return run(node_cmd)
 
 
+def go_net_http_benchmark():
+    # Could not easily track process created by simply "go run"
+    tmp_prog = tempfile.mkdtemp() + "/go_net_http" + (".exe" if os.name == "nt"
+                                                      else "")
+    go_build_cmd = ["go", "build", "-o", tmp_prog, "tools/go_net_http.go"]
+    subprocess.call(go_build_cmd)
+    print "http_benchmark testing GO net/http."
+    return run([tmp_prog, ADDR.split(":")[1]])
```

Comment:
Adding Go as a dep is pretty heavy - it's a whole new runtime that we don't use currently. Could you instead use a hyper http server - EG https://github.com/hyperium/hyper/blob/master/examples/hello.rs
You can add a new `rust_binary("hyper_hello")` to //BUILD.gn 
In both rust and go, they are going to be an order of magnitude or more faster than we can ever achieve with a single threaded dynamic runtime. However it will provide something to strive for.

Reply:
Okay